packages/widget/src/hooks/use-chains.ts

```diff
@@ -32,8 +31,8 @@ export function useChains<T = Chain[]>(args: UseChainsQueryArgs<T> = {}) {
         .map((chain): Chain => {
           return {
             ...chain,
-            chainName: chainIdToName[chain.chainID] || chain.chainName,
-            prettyName: chainIdToPrettyName[chain.chainID] || chain.chainName,
+            chainName: chain.chainName || getChain(chain.chainID).chain_name,
+            prettyName: chain.prettyName || getChain(chain.chainID).pretty_name,
```

Comment:
@codingki I don't know if `chain.chainName` and  `getChain(chain.chainID).chain_name` can both exist
but `getChain(chain.chainID).chain_name || chain.chainName` is what we had before, this looks like you flipped the priority of chain.chainName to be first

Reply:
yes intentional. We are trying to not use `getChain`. but will have a different PR for that task

Reply:
ok, sounds good